packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -98,12 +101,18 @@ function isDeposit(tx: DepositOrWithdrawal): tx is Deposit {
   return tx.direction === 'deposit'
 }
 
-async function transformTransaction(
-  tx: DepositOrWithdrawal
-): Promise<MergedTransaction> {
+export function isCctpTransfer(tx: Transfer): tx is MergedTransaction {
+  return !!(tx as MergedTransaction).isCctp
+}
+
+async function transformTransaction(tx: Transfer): Promise<MergedTransaction> {
   const parentChainProvider = getProvider(tx.parentChainId)
   const childChainProvider = getProvider(tx.childChainId)
 
+  if (isCctpTransfer(tx)) {
+    return tx
+  }
```

Comment:
Maybe the base case can just be to return `tx` instead of throwing the error?

Reply:
We need the return type to be `MergedTransaction`, but tx prop can be a partial transaction that will be transformed, so we cannot just return it as a base case.

We can return tx for cctp transfers because they are already of type `MergedTransaction`.